packages/gatsby-source-sql/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "gatsby-source-sql",
```

Comment:
you should rename it to be a vendor specific package: `gatsby-source-mysql` there are a lot of sql variations and you don't want them all in one package